src/minErr.js

```diff
@@ -34,19 +34,32 @@ function minErr(module, ErrorConstructor) {
   ErrorConstructor = ErrorConstructor || Error;
   return function() {
     var SKIP_INDEXES = 2;
+    var DEFAULT_MAX_DEPTH = 3;
 
     var templateArgs = arguments,
+      argsDebuggedStringIndexes = [],
       code = templateArgs[0],
       message = '[' + (module ? module + ':' : '') + code + '] ',
       template = templateArgs[1],
       paramPrefix, i;
 
-    message += template.replace(/\{\d+\}/g, function(match) {
-      var index = +match.slice(1, -1),
-        shiftedIndex = index + SKIP_INDEXES;
+    function getArgumentAndConvertToDebugString(index, maxDepth) {
```

Comment:
@gkalpak Thanks for helping, 
I forget to add configureMinErr in test/.eslintrc.json so i couldn't use inside the tests,

I have fixed all things that you left in your comments.
Please tell me if there is something else missing.